package/ios/Core/Utils/ModernCameraOrientationCoordinator.swift

```diff
@@ -14,18 +14,20 @@ class ModernCameraOrientationCoordinator: CameraOrientationCoordinator {
   private weak var delegate: CameraOrientationCoordinatorDelegate?
   private var previewObserver: NSKeyValueObservation?
   private var outputObserver: NSKeyValueObservation?
+  private var device: AVCaptureDevice?
```

Comment:
```suggestion
  private let device: AVCaptureDevice
```

Device cannot be nil here.